modules/deploy/partials/high-availability.adoc

```diff
@@ -112,7 +112,7 @@ endif::[]
 A multi-region deployment is similar to a multi-AZ deployment, in that it needs at least three regions to counter the loss of a single region. Note that this deployment strategy increases latency due to the physical distance between regions. Consider the following strategies to mitigate this problem:
 
 * Manually configure leadership of each partition to ensure that leaders are congregated in the primary region (closest to the producers and consumers).
-* Configure producers to have `acks=1` instead of `acks=all`; however, this introduces the possibility of losing messages if the primary region becomes lost or unavailable.
+* If your produce latency exceeds your tolerances, you can configure producers to have `acks=1` instead of `acks=all`; however, this introduces the possibility of losing messages if the primary region becomes lost or unavailable.
```

Comment:
How about this: "If your produce latency is higher than you require, you could configure producers to use `acks=1` instead of `acks=all` - this is faster, since the leader can acknowledge immediately without waiting for the other brokers. Keep in mind that this reduces message durability: if the partition leader goes offline, any messages that are acknowledged but not yet replicated may be lost."

Reply:
Reworded it along these lines.